tools/run_tests/run_xds_tests.py

```diff
@@ -109,6 +110,10 @@ def parse_port_range(port_arg):
     default='',
     help='File to reference via GRPC_XDS_BOOTSTRAP. Disables built-in '
     'bootstrap generation')
+argp.add_argument(
+    '--xds_v3_support',
+    default=False,
```

Comment:
I think this needs `action='store_true',` as with the other boolean flags.

Reply:
Done.